sdk/management/docs/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,284 @@
+## Guide for migrating to `com.azure.resourcemanager.**` from `com.microsoft.azure-mgmt-**`
```

Comment:
I think this is refer to namespace, it should be `com.microsoft.azure.management.**` for track 1

Reply:
fixed